Source/LinkedInWebViewController.swift

```diff
@@ -49,7 +47,14 @@ class LinkedInWebViewController: UIViewController, UIWebViewDelegate {
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        webView = UIWebView(frame: view.bounds)
```

Comment:
Please use autolayout to do this. 
```
       let layoutViews: [String: Any] = ["view": self.view, "webview": webview]
        let horizontalConstraints = NSLayoutConstraint.constraints(
            withVisualFormat: "H:|-[segmentView]-|",
            options: NSLayoutFormatOptions.alignAllCenterY,
            metrics: nil, views: layoutViews)
        view.addConstraints(horizontalConstraints)

        let veriticalConstraints = NSLayoutConstraint.constraints(
            withVisualFormat: "V:|-[segmentView]-|",
            options: NSLayoutFormatOptions.alignAllCenterY,
            metrics: nil, views: layoutViews)
        view.addConstraints(veriticalConstraints)

```
![simulator screen shot jun 7 2017 10 02 25 am](https://user-images.githubusercontent.com/1928509/26882680-0991a7d4-4b69-11e7-948d-7293c580e0d5.png)